__tests__/schema.test.js

```diff
@@ -0,0 +1,54 @@
+//const Schema = require('../lib/schema.js');
+const Schema = require('../lib/schema');
```

Comment:
capitalize the S in the path
old: const Schema = require('../lib/schema');
new: const Schema = require('../lib/Schema');